packages/open-api-page/partials/two-column-layout/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import s from './style.module.css'
+
+function TwoColumnLayout({
+  columnOne,
+  columnTwo,
+}: {
+  /** Render slot for the first column. */
+  columnOne: React.ReactNode
+  /** Render slot for the second column. */
+  columnTwo: React.ReactNode
+}): React.ReactElement {
+  return (
+    <div className={s.twoColumnLayout}>
+      <div>{columnOne}</div>
+      <div></div>
```

Comment:
nit: Is this just for spacing? could we use margin/padding on the first or second element instead?